CITATION.cff

```diff
@@ -25,22 +25,11 @@ authors:
   given-names: Ben
 - family-names: Nouvellet
   given-names: Pierre
-repository-code: https://github.com/epiverse-trace/serofoi
-url: https://epiverse-trace.github.io/serofoi/
```

Comment:
Is there a specific reason these URLs were removed from `DESCRIPTION` in https://github.com/epiverse-trace/serofoi/commit/c54ebfe39a9a62437938655744bca2ad3b85c547 or was this an accident?